R/description.R

```diff
@@ -480,8 +482,8 @@ description <- R6Class("description",
       idesc_get(self, private, keys),
 
     get_field = function(key, default = stop("Field '", key, "' not found"),
-                         trim_ws = TRUE)
-      idesc_get_field(self, private, key, default, trim_ws),
+                         trim_ws = TRUE, squish_ws = trim_ws)
```

Comment:
Setting the default to `TRUE` is a breaking change, unfortunately.

Reply:
I know. But isn't that the behaviour most people actually need, i.e. would expect without prior knowledge...?

Do you want me to change the default to `FALSE`?

Reply:
I think for many fields, most people want the specialized accessors, i.e. `get_deps()` instead of manually parsing `Includes`, etc.

But yeah, maybe we can keep the `TRUE`.

Reply:
I've explicitly mentioned it as a "breaking change" in the NEWS 😇